WLogger/WeekLogs/WeekLogsTableViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  WeekLogsTableViewController.swift
+//  WLogger
+//
+//  Created by Pablo Giorgi on 1/10/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class WeekLogsTableViewController: UITableViewController {
+
+    var viewModel: WeekLogsTableViewModel?
+}
+
+extension WeekLogsTableViewController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.refreshControl = UIRefreshControl()
+        self.tableView.addSubview(refreshControl!)
+        self.refreshControl?.addTarget(self, action: "handleRefresh:", forControlEvents: UIControlEvents.ValueChanged)
+        tableView.registerNib(UINib(nibName: "LogTableViewCell", bundle: nil), forCellReuseIdentifier: LogTableViewCellIdentifier)
```

Comment:
OK, I didn't like to do this. I really don't know why it uses a String, when in objc it used a selector, which is much better. Anyway, I'd create an identifier in this class, since this can be encapsulated in this same class.